Social/Twitter/Oauth.php

```diff
@@ -0,0 +1,156 @@
+<?php
```

Comment:
All the comment from "Social/Facebook/Oauth.php" also apply to this file.
Added to that is when there is no request there is also no session.